sources/packages/backend/apps/db-migrations/src/sql/Reports/Create-institution-designation-report.sql

```diff
@@ -0,0 +1,41 @@
+INSERT INTO
+  sims.report_configs (report_name, report_sql)
+VALUES
+  (
+    'Institution_Designation_Report',
+    'SELECT
+      institutions.operating_name AS "Institution Operating Name",
+      institution_locations.name AS "Location Name",
+      institution_locations.institution_code AS "Location Code",
+      institution_type.name AS "Institution Type",
+      designation_agreements.designation_status AS "Designation Status",
+      CAST(
+        CAST(designation_agreements.assessed_date AS date) AS varchar
+      ) AS "Assessed Date",
+      CAST(designation_agreements.end_date AS varchar) AS "Expiry Date",
+      CONCAT(
+        institution_locations.primary_contact ->> ''firstName'',
+        '' '',
+        institution_locations.primary_contact ->> ''lastName''
+      ) AS "Location Contact",
+      designation_agreement_locations.requested AS "Request for designation",
+      designation_agreement_locations.approved AS "Approved for designation",
```

Comment:
I do not see a reason for "Request for designation" and "Approved for designation" be in the middle of the "Location Contact" and "Contact Email".
Can you please move "Location Contact" right above "Contact Email"?